packages/diagrams/frontend/sirius-components-diagrams/src/sprotty/views/InsideLabelView.tsx

```diff
@@ -84,6 +85,36 @@ export class InsideLabelView extends SLabelView {
         {label.text}
       </div>
     );
+
+    const debugFontSize = 6;
+    const styleDebug = {
+      fontSize: debugFontSize + 'px',
+      fill: 'red',
+    };
+
+    const debugInfos = (
+      node: { root: Diagram; bounds: { x: number; y: number } },
+      size: { width: number; height: number }
+    ) => {
+      const diagram: Diagram = node.root as Diagram;
+      if (diagram.debug) {
+        return (
+          <g>
+            <text x={size.width / 2} y={0 - debugFontSize * 2} style={styleDebug}>
+              x:{node.bounds.x.toFixed(2)} y:{node.bounds.y.toFixed(2)}
+            </text>
+            <text x={size.width} y={size.height - debugFontSize} style={styleDebug}>
+              Width: {size.width.toFixed(2)}
+            </text>
+            <text x={0} y={size.height - debugFontSize} style={styleDebug}>
+              Height: {size.height.toFixed(2)}
+            </text>
+            {debugInfos(label, label.size)}
+          </g>
+        );
+      } else return null;
+    };
+
```

Comment:
You do not need to duplicate that much code and on top of that this code is unused